modules/media/jsx/mediaIndex.js

```diff
@@ -0,0 +1,32 @@
+/* global DynamicDataTable MediaUploadForm formatColumn */
+
+$(function() {
+  loris.hiddenHeaders = ['Cand ID', 'Session ID'];
```

Comment:
There shouldn't be spaces in CandID and SessionID to be consistent with the rest of LORIS